volcano/map.conf

```
@@ -0,0 +1,34 @@
+license = CC BY-SA 4.0
+
+author = ksandr
+name = Volcano
+
+h = 140
+r = 115
+rotation = z
+
+initial_stuff = grenades:smoke,shooter:pistol,default:wood 50,default:ladder_wood 50,default:axe_steel,default:torch 10
+base_node = default:steelblock
+
+# this im copy from tunnel map (i like crossbow)
+treasures = default:ladder,0.3,2,10,20;default:glass,0.3,2,5,15;default:cobble,0.4,5,45,99;default:stone,0.2,3,10,20;default:wood,0.3,5,30,60;default:gravel,0.3,3,10,20;default:obsidian,0.06,2,3,6;default:steel_ingot,0.15,2,5,10;default:mese_crystal,0.08,2,2,4;default:diamond,0.05,2,1,2;doors:door_steel,0.3,5,1,3;ctf_traps:damage_cobble,0.3,4,10,20;ctf_map:reinforced_cobble,0.3,4,10,20;default:sword_mese,0.1,3,1,2;default:sword_diamond,0.06,2,1,2;default:shovel_mese,0.1,3,1,2;default:shovel_diamond,0.06,2,1,2;default:axe_steel,0.15,2,1,2;default:axe_mese,0.1,3,1,2;default:axe_diamond,0.06,2,1,2;default:pick_diamond,0.1,2,1,2;shooter:shotgun,0.05,2,1;grenades:frag,0.1,2,1;grenades:smoke,0.1,2,1;shooter:machine_gun,0,03,2,1;shooter:crossbow,0.2,2,1,2;shooter:pistol,0.4,2,1,4;shooter:ammo,0.3,2,1,10;shooter:arrow_white,0.3,2,10,20;medkits:medkit,0.8,5,2;ctf_bandages:bandage,0.8,2,3,6
```

Well, I thought you liked sniper rifles ;)

```suggestion
treasures = default:ladder,0.3,2,10,20;default:glass,0.3,2,5,15;default:cobble,0.4,5,45,99;default:stone,0.2,3,10,20;default:wood,0.3,5,30,60;default:gravel,0.3,3,10,20;default:obsidian,0.06,2,3,6;default:steel_ingot,0.15,2,5,10;default:mese_crystal,0.08,2,2,4;default:diamond,0.05,2,1,2;doors:door_steel,0.3,5,1,3;ctf_traps:damage_cobble,0.3,4,10,20;ctf_map:reinforced_cobble,0.3,4,10,20;default:sword_mese,0.1,3,1,2;default:sword_diamond,0.06,2,1,2;default:shovel_mese,0.1,3,1,2;default:shovel_diamond,0.06,2,1,2;default:axe_steel,0.15,2,1,2;default:axe_mese,0.1,3,1,2;default:axe_diamond,0.06,2,1,2;default:pick_diamond,0.1,2,1,2;shooter:shotgun,0.05,2,1;grenades:frag,0.1,2,1;grenades:smoke,0.1,2,1;shooter:machine_gun,0,03,2,1;shooter:crossbow,0.2,2,1,2;shooter:pistol,0.4,2,1,4;shooter:ammo,0.3,2,1,10;shooter:arrow_white,0.3,2,10,20;medkits:medkit,0.8,5,2;sniper_rifles:rifle_762_loaded,0.15,2,1;sniper_rifles:rifle_magnum_loaded,0.04,2,1
```